itest/rust/src/engine_tests/match_class_test.rs

```diff
@@ -22,19 +22,20 @@ fn match_class_basic_dispatch() {
     let to_free = obj.clone();
 
     let result = match_class! { obj,
+        _ @ Node3D => 3,
         node @ Node2D => {
             require_node2d(&node);
-            1
+            2
         },
         node @ Node => {
             require_node(&node);
-            2
+            1
         },
-        _ => 3 // No comma.
+        _ => 0 // No comma.
```

Comment:
Minor detail, but why the change in enumeration? Keep it ascending from 1 onward.